packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -24,6 +30,38 @@ const assertServiceHealthy = async (apiUrl: string, serviceName: ServiceNames) =
   expect(res.data).toEqual({ ok: true });
 };
 
+type CallCliAndAssertExitArgs = {
+  expectedExitCode: number;
+  assertionsCount: number;
+  dataMatchOnError?: string;
+  args?: string[];
+  env?: NodeJS.ProcessEnv;
+};
+
+const callCliAndAssertExit = (
+  { expectedExitCode, args = [], env = {}, dataMatchOnError, assertionsCount }: CallCliAndAssertExitArgs,
+  done: jest.DoneCallback
+) => {
+  const spy = jest.fn();
+  expect.assertions(assertionsCount);
```

Comment:
```suggestion
  expect.assertions(dataMatchOnError ? 3 : 2 );
```
The expected assertions are deterministic right? Best to keep this an internal concern

Reply:
Done in b2ff0d7cc6e56dda7c265fdb4195c0e1080975e4